XenAdmin/Diagnostics/Checks/PBDsPluggedCheck.cs

```diff
@@ -40,10 +40,17 @@ namespace XenAdmin.Diagnostics.Checks
 {
     public class PBDsPluggedCheck : Check
     {
+        public List<SR> srUploadedUpdates = new List<SR>();

```

Comment:
`srUploadedUpdates  `should be private here, as it is only used in this class and set in the constructor.